src/components/VoiceCompair/VoiceCompair.js

```diff
@@ -109,6 +110,33 @@ const VoiceCompair = props => {
       body: payload,
       redirect: 'follow',
     };
+    function compareArrays(arr1, arr2) {
```

Comment:
can we move this compareArrays logic also in one place?

Reply:
Yes, it is moved now.